<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabwidgetmain">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_raw_data">
       <attribute name="title">
        <string>raw data</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView_raw"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_treated_data">
       <attribute name="title">
        <string>treated data</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView_treated"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="action_file_project_new"/>
    <addaction name="action_file_project_open"/>
    <addaction name="action_file_project_save"/>
    <addaction name="action_file_project_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_file_exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_view_ToggleFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="action_view_LastPerspective"/>
    <addaction name="action_view_OpenOtherPerspective"/>
    <addaction name="action_view_FactoryPerspective"/>
    <addaction name="action_view_saveCurrentPerspective"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockwidget1">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>166</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string notr="true">Pro&amp;ject</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidgetProject">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget2">
   <property name="minimumSize">
    <size>
     <width>319</width>
     <height>410</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string notr="true">Setti&amp;ngs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabSettings_select">
        <attribute name="title">
         <string>select</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox_select1"/>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_select1">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_select2"/>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_select2">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_select3"/>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_select3">
           <property name="selectionMode">
            <enum>QAbstractItemView::ContiguousSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSettings_treat">
        <attribute name="title">
         <string>treatment</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>selected</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_treatment_add">
           <property name="text">
            <string>add &gt;</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QListWidget" name="listWidget_treatSelected"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pretreatments</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QListWidget" name="listWidget_treatAvailable"/>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_treatment_remove">
           <property name="text">
            <string>&lt; remove</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>available</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_treatment_apply">
           <property name="text">
            <string>apply immediately</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_treatment_apply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>apply now</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSettings_plot">
        <attribute name="title">
         <string>plot</string>
        </attribute>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>colour defined by:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboPlot_ColourBy">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>241</width>
           <height>33</height>
          </rect>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidgetPlot_ColourBy">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>241</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_file_project_new">
   <property name="text">
    <string>&amp;new Project</string>
   </property>
  </action>
  <action name="action_file_project_open">
   <property name="text">
    <string>&amp;open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_file_project_save">
   <property name="text">
    <string>&amp;save project</string>
   </property>
  </action>
  <action name="action_file_project_saveas">
   <property name="text">
    <string>save &amp;project as</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="action_view_ToggleFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_view_LastPerspective">
   <property name="text">
    <string>last Perspective</string>
   </property>
  </action>
  <action name="action_view_OpenOtherPerspective">
   <property name="text">
    <string>open other Perspective</string>
   </property>
  </action>
  <action name="action_view_FactoryPerspective">
   <property name="text">
    <string>Factory Perspective</string>
   </property>
  </action>
  <action name="action_view_saveCurrentPerspective">
   <property name="text">
    <string>save current Perspective</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="action_file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_file_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
